satellite-placement/README.rst

```diff
@@ -0,0 +1,63 @@
+Satellite Placement Demo
+========================
+Suppose you have a set of ``N`` satellites and ``k`` targets on Earth that you
+want to observe. Each of your satellites has varying capabilities for Earth
+observation; in particular, the amount of ground that they can observe for a
+set amount of time is different. Since there are ``k`` targets, you would like
+to have ``k`` constellations to monitor said targets. How do you group your
+satellites into ``k`` constellations such that the average coverage of each
+constellation is maximized? This is the question that we will be addressing in
+this demo!
+
+Note: in this demo we are assuming that ``N`` is a multiple of ``k``.
+
+Usage
+-----
+To run the demo,
+::
+  python satellite.py
+
+It will print out a set of satellite constellations.
+
+Code Overview
+-------------
+The idea is to consider all possible combinations of satellites, eliminate
+constellations with particularly low coverage, and encourage certain
+properties in our final solution. Namely, we want to encourage:
```

Comment:
"coverage, and encourage certain properties in our final solution. Namely, we want to encourage:" --> Might be easier without the temporary abstraction, "properties"; something like:

"...coverage, and encourage the following type of solutions:"